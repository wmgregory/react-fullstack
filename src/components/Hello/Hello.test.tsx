import * as React from 'react';
import { shallow } from 'enzyme';

import { Hello } from './Hello';

describe('Hello', () => {
  it('should exist', () => {
    const wrapper = shallow(<Hello compiler="test" framework="test" />);

    expect(wrapper).toMatchSnapshot();
  });
});
