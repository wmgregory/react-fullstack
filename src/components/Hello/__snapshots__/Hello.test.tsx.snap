// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello should exist 1`] = `
<h1>
  Hello from 
  test
   and 
  test
  !
</h1>
`;
